apps/server/src/shared/repo/base.do.repo.ts

```diff
@@ -17,11 +17,17 @@ export abstract class BaseDORepo<DO extends BaseDO, E extends BaseEntity, P> {
 	protected abstract mapDOToEntityProperties(entityDO: DO): P;
 
 	async save(entityDo: DO): Promise<DO> {
-		const savedDos: DO[] = await this.saveAll([entityDo]);
+		const savedDos: DO[] = await this.saveAllWithoutFlush([entityDo]);
+		await this._em.flush();
+		return savedDos[0];
+	}
+
+	async saveWithoutFlush(entityDo: DO): Promise<DO> {
```

Comment:
when you remove the transaction, please undo this as well.

Reply:
done